charts/opensearch/ci/ci-values.yaml

```diff
@@ -320,12 +322,20 @@ terminationGracePeriod: 120
 
 sysctlVmMaxMapCount: 262144
 
+startupProbe:
```

Comment:
I think all these overrides should be removed so that the actual chart defaults are tested in CI. i.e. remove the `startupProbe` and `readinessProbe` blocks from this file entirely.

What do you think?

Reply:
In one of the CI runs we can remove, that should be fine

Reply:
We can do that